models/quality_measures/intermediate/cbe0055_diabetes_eye_exam/quality_measures__int_cbe0055__denominator.sql

```diff
@@ -0,0 +1,208 @@
+{{ config(
+     enabled = var('quality_measures_enabled',var('claims_enabled',var('clinical_enabled',var('tuva_marts_enabled',False)))) | as_bool
+   )
+}}
+
+with visit_codes as (
+
+    select
+          code
+        , code_system
+    from {{ ref('quality_measures__value_sets') }}
+    where lower(concept_name) in (
+          'office visit'
+        , 'home healthcare services'
+        , 'preventive care services established office visit, 18 and up'
+        , 'preventive care services initial office visit, 18 and up'
+        , 'annual wellness visit'
+        , 'telephone visits'
+        , 'emergency department evaluation and management visit'
```

Comment:
Remove this concept from the denominator visit_codes logic.

Reply:
Can you please share the reason as to why they're removed? This is in the `value set concepts` seeds for this measure.

Reply:
@utsavpaudel sorry for the confusion! Only 'emergency department evaluation and management visit' should be removed from the list. This concept should not be used in the denominator. It's only used for the acute inpatient exclusion logic.